src/Templater.js

```diff
@@ -89,13 +89,17 @@ export default class Templater {
      *      // where one of these countries can be null
      *      templater.derive_language(user_country, channel_country, signup_country)
      */
-    derive_language (...keys) {
+    derive_language(...keys) {
+
+        const find = key => {
 
-        const find = key =>
-            language_group =>
-                _.includes(language_group.countries, key)
-                || language_group.countries[key];
+            const country_code = _.upperCase(key);
+            const country_name = _.startCase(_.lowerCase(key));
```

Comment:
startCase should already taken care of the lowerCase
https://lodash.com/docs/4.17.4#startCase
https://en.wikipedia.org/wiki/Letter_case#Stylistic_or_specialised_usage

Reply:
I don't think so.
https://jsfiddle.net/ferrielmelarpis/ytafe24o/
Also, if you check the implementation, it doesn't apply lowercase to trailing letters of a word.
https://github.com/lodash/lodash/blob/462bc9bf3b0ac2738098d3045e80a2a8910203de/startCase.js#L24-L28

Reply:
Okay, guess I read it wrong, thought all cap is the input in the wiki's example.